src/material-experimental/mdc-form-field/form-field.ts

```diff
@@ -340,7 +340,15 @@ export class MatFormField implements AfterViewInit, OnDestroy, AfterContentCheck
     this._updateFocusState();
     // Initial notch width update. This is needed in case the text-field label floats
     // on initialization, and renders inside of the notched outline.
-    this._refreshOutlineNotchWidth();
+    if (this._platform.TRIDENT) {
```

Comment:
Rather than looking at the browser, you should be able to detect the feature instead. E.g. `if (document.fonts && document.fonts.ready)`. Usually we only do browser detection as a last resort when feature detection isn't possible.